### PR TITLE
Changes to enable @CEE for T211125

### PR DESCRIPTION
**Pull request by hknustwmf**

### CODE REVIEW

**Comment by Pchelolo** [comment]
oh, actually, could you also explicitly bump the syslog-udp dependency version

File: lib/logger.js [hunk 1]
```diff
@@ -70,7 +70,8 @@ const streamConverter = {
         const impl = syslogStream.createBunyanStream({
             facility: stream.facility || 'local0',
             host: stream.host || '127.0.0.1',
-            port: stream.port || 514
+            port: stream.port || 514,
+            prefix: stream.prefix || ''
```
Comment (Pchelolo) [inline lib/logger.js]: The package also allows to set up the 'name'. Let's change this from explicit properties remapping to an `Object.assign` that would provide the defaults for the host, port and facility in order to allow additional configuration properties to be passed as is.

Comment (hknustwmf) [reply lib/logger.js]: Done

File: lib/logger.js [hunk 2]
```diff
@@ -67,11 +67,14 @@ const streamConverter = {
...
-            port: stream.port || 514
-        });
+        const defaultOpts = {
+            facility: 'local0',
+            host: '127.0.0.1',
+            port: 514,
+            prefix: ''
+        };
+
+        const impl = syslogStream.createBunyanStream(Object.assign(stream, defaultOpts));
```
Comment (Pchelolo) [inline lib/logger.js]: This should be other way around. The right argument overrides the left argument.

**Pull request merged**